gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_5_0/01_add_async_jobs_table.yml

```diff
@@ -1,14 +1,15 @@
 databaseChangeLog:
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.

Reply:
As 4.5 is not released it is ok to update the existing changelog